packages/typescript/ai-client/src/chat-client.ts

```
@@ -11,6 +11,8 @@ import type {
   StreamChunk,
 } from '@tanstack/ai'
 import type { ConnectionAdapter } from './connection-adapters'
+import type { SessionAdapter } from './session-adapter'
+import { createDefaultSession } from './session-adapter'
```

ESLint

[error] `./session-adapter` import should occur before type import of `@tanstack/ai`

(import/order)